HelloWorld/HelloWorld/ViewModel/MainViewModel.cs

```diff
@@ -39,14 +39,14 @@ public ICommand GreetCommand
         /// </summary>
         public MainViewModel()
         {
-            ////if (IsInDesignMode)
-            ////{
-            ////    // Code runs in Blend --> create design time data.
-            ////}
-            ////else
-            ////{
-            ////    // Code runs "for real"
-            ////}
+            if (IsInDesignMode)
+            {
+                // Code runs in Blend --> create design time data.
+            }
+            else
+            {
+                // Code runs "for real"
+            }
```

Comment:
Why not delete at all? Code does nothing

Reply:
We will need that, because ...

Reply:
Ah, I understand. That's fine